packages/core/src/components/ui/Checkbox/Checkbox.js

```diff
@@ -55,34 +57,36 @@ const Checkbox = forwardRef(function Checkbox(props, ref) {
   }
 
   return (
-    <div
-      ref={ref}
-      className={classes}
-      htmlFor={id}
-      tabIndex={disabled ? -1 : 0}
-      role='checkbox'
-      aria-checked={checked}
-      onClick={handleLabelClick}
-      onKeyPress={utils.handleKeyboardEvent(' ', handleLabelClick)}
-      // eslint-disable-next-line react/jsx-props-no-spreading
-      {...rest}
-    >
-      {/* eslint-disable-next-line react/jsx-props-no-spreading */}
-      <input {...inputProps} />
+    <Tooltip content={customhelp}>
```

Comment:
can you please change to following to avoid route to Tooltip comp when customhelp is not given
`const content = ...
if(!_isEmpty(customhelp)) {
  return <Tooltip>content</Tooltip>
}

return content

`